sos/report/plugins/nvidia.py

```diff
@@ -23,13 +23,22 @@ def setup(self):
             '--list-gpus',
             '-q -d PERFORMANCE',
             '-q -d SUPPORTED_CLOCKS',
-            '-q -d PAGE_RETIREMENT'
+            '-q -d PAGE_RETIREMENT',
+            '-q',
+            '-q -d ECC',
+            'nvlink -s',
+            'nvlink -e'
         ]
 
         self.add_cmd_output(["nvidia-smi %s" % cmd for cmd in subcmds])
 
         query = ('gpu_name,gpu_bus_id,vbios_version,temperature.gpu,'
-                 'utilization.gpu,memory.total,memory.free,memory.used')
+                 'utilization.gpu,memory.total,memory.free,memory.used'
+                 'clocks.applications.graphics,clocks.applications.memory')
+        querypages = ('timestamp,gpu_bus_id,gpu_serial,gpu_uuid,'
+                       'retired_pages.address,retired_pages.cause')
         self.add_cmd_output("nvidia-smi --query-gpu=%s --format=csv" % query)
+        self.add_cmd_output("nvidia-smi --query-gpu=%s --format=csv" % query)
+        self.add_cmd_output("nvidia-smi ----query-retired-pages=%s --format=csv" % querypages)
```

Comment:
Line is too long, causing a flake8 error, please wrap accordingly.

```python
self.add_cmd_output(
    "nvidia-smi ----query-retired-pages=%s --format=csv" % querypages
)
```

should work